nvflare/app_common/workflows/model_controller.py

```diff
@@ -138,7 +138,9 @@ def start_controller(self, fl_ctx: FLContext) -> None:
         else:
             self.model = FLModel(params_type=ParamsType.FULL, params={})
 
-        self.fl_ctx.set_prop(AppConstants.GLOBAL_MODEL, self.model, private=True, sticky=True)
+        # persistor uses Learnable format to save model
+        ml = make_model_learnable(weights=self.model.params, meta_props=self.model.meta)
+        self.fl_ctx.set_prop(AppConstants.GLOBAL_MODEL, ml, private=True, sticky=True)
```

Comment:
just a note, we might need to think in 2.5 that what format should persistor take in.